forgerock-auth/src/main/java/org/forgerock/android/auth/SessionManager.java

```diff
@@ -97,6 +97,10 @@ void close() {
         singleSignOnManager.revoke(null);
     }
 
+    void revokeAccessToken(FRListener<Void> listener) {
```

Comment:
Add javadoc to be consistent with previous methods within the class

Reply:
done